sdk/src/clearingHouseUser.ts

```diff
@@ -172,6 +176,103 @@ export class ClearingHouseUser {
 		return userAccountRPCResponse.value !== null;
 	}
 
+	// TODO
+	// /**
+	//  * calculates the market position if the lp position was settled
+	//  * @returns : userPosition
+	//  */
+	// public getSettledLPPosition(marketIndex: BN): UserPosition {
```

Comment:
lets get working version of `getSettledLPPosition`, so user can see what they'd settle on ui?